bosh/secrets.example.yml

```diff
@@ -34,12 +33,40 @@ properties:
     component_name: cf-swarm-containers-broker
     docker_url: secret
 
-
 networks:
 - name: default
-  type: dynamic
+  type: manual
```

Comment:
I tried using two `network`s and wound up hitting timeouts on deploy, which is why I went this direction. But there's a non-trivial chance that I just set it up wrong, so we can take another look if you're interested.